pkg/conthandler/container_watcher.go

```diff
@@ -149,6 +161,10 @@ func (containerWatcher *ContainerWatcher) StartWatchedOnContainers(containerEven
 							continue
 						}
 						containerEventChannel <- *containerEventData
+					} else if pod.Status.ContainerStatuses[i].State.Terminated != nil {
+						logger.L().Debug("container has Terminated", helpers.String("namespace", pod.GetNamespace()), helpers.String("Pod name", pod.GetName()), helpers.String("ContainerID", pod.Status.ContainerStatuses[i].ContainerID), helpers.String("Container name", pod.Status.ContainerStatuses[i].Name))
```

Comment:
I'm not sure we want a debug log dor each container that is terminating. What do you think?